docs/sources/datasources/_index.md

```diff
@@ -92,8 +92,8 @@ These built-in core data sources are also included in the Grafana documentation:
 [alerts]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/alerting"
 [alerts]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/alerting"
 
-[build-a-plugin]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins"
-[build-a-plugin]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins"
+[build-a-plugin]: "/docs/grafana/ -> /developers/plugin-tools/"
```

Comment:
Reviving https://github.com/grafana/grafana/pull/75325#discussion_r1336212201.

All these links should to `/developers/` documentation should be replaced with inline links instead of using the `docs/reference` shortcode. The shortcode checks links with the `relref` function internally which will always error since these pages are not part of the build. The return value of the erroring `relref` function is a 404 page which is not what you want.